client/my-sites/site-logs/main.tsx

```diff
@@ -52,8 +65,20 @@ export function SiteLogs() {
 				subHeaderText={ __( 'View server logs to troubleshoot or debug problems with your site.' ) }
 				align="left"
 			/>
-			<h2>Web logs</h2>
-			<pre>{ formattedLogs }</pre>
+
+			<SiteLogsTabPanel
+				initialTab={ tab }
+				onSelected={ ( tabName ) => {
+					handleTabSelected( tabName );
+				} }
```

Comment:
By passing an anonymous function to the `onSelected` prop we lose the advantage of `useCallback` because the prop value will change each render anyway. It's not a big deal because this probably isn't a performance bottleneck anyway, but we should either remove the `useCallback` or just pass `handleTabSelected` directly to `onSelected`

```suggestion
				onSelected={ handleTabSelected }
```